.github/CODEOWNERS

```diff
@@ -336,7 +336,6 @@
 
 
 # Navigation bar project.
-/core/templates/domain/sidebar/ @nithusha21
```

Comment:
From which line this got replaced? do we need to assign the new file to someone?

Reply:
Only Line 339 was removed and I don't see any need to assign new directory to anyone:

/core/templates/domain/sidebar/ - No Longer Exist.

/core/templates/services/ - Directory where the file was relocated already has many files in it and also codeowner assigned to the directory.

@DubeySandeep PTAL! ( Also you tagged wrong one, its @HarshKhilawala )

Reply:
> /core/templates/services/ - Directory where the file was relocated already has many files in it and also some codeowners assigned to different files in the directory. I will assign @nithusha21 to this file.

Sounds good to me!